### ISSUE TITLE
[Inspect]There is no "Font" edit item below "Font" tree item node when Font property is Expanded

### ISSUE THREAD

**Issue by Marie-Shi** [dotnet/winforms#2848]
-  .NET Core Version: .Net Core 3.1.200-preview-014978 from 3.1 Release branch + VS2019 with latest core designer
- Have you experienced this same bug with .NET Framework?: Yes

**More Info:**
1.	This issue can reproduce on Font, Location, Margin, Size… properties.
2.	This issue can also reproduce on Framework project with latest .Net 4.8.
3.	Using Inspect to focus Font property, there is "Font" edit item in the first time for collapse status, no "Font" edit item in the second time. Please see the result of Step5.

**Steps to reproduce:**
1.	Open the attached “Font_Inspect_issue.zip” in VS. 
[Font_Inspect_issue.zip](https://github.com/dotnet/winforms/files/4191735/Font_Inspect_issue.zip)
2.	Build and run this application, select the PropertyGrid control. 
3.	Open Inspect and focus on the Font property, observe the inspect behavior. 
4.	Expand the Font property and observe the inspect behavior. 
5.	Collapse the Font property and observe the inspect behavior.

**Actual behavior:**
After step 3, there is "Font" edit item below "Font" tree item node.
![Step3](https://user-images.githubusercontent.com/45864017/74330482-c01cc700-4dcc-11ea-8052-0da3c7de94bd.png)
After step 4, there is no "Font" edit item below "Font" tree item node.
![Step4](https://user-images.githubusercontent.com/45864017/74330495-c612a800-4dcc-11ea-9bb5-530f39b25ceb.png)
After step 5, there is no "Font" edit item below "Font" tree item node. Please see the Gif:
![Font_expand_collapse_issue](https://user-images.githubusercontent.com/45864017/74330551-d88ce180-4dcc-11ea-80c0-cec04ee5a59e.gif)

**Expected behavior:**
After step 4, there should be "Font" edit item below "Font" tree item node.
After step 5, there should be "Font" edit item below "Font" tree item node, the result should be same as step3.

**Comment by Amy-Li03** [comment]
Verified this issue on .NET 7.0.100-preview.5.22226.4 build, issue was fixed. 
After step 4, there is "Font" edit item below "Font" tree item node.
![image](https://user-images.githubusercontent.com/62929087/165665219-b4e9ae39-23e7-4730-bf6d-b338b1a8ec3c.png)
After step 5, there is also "Font" edit item below "Font" tree item node.
![image](https://user-images.githubusercontent.com/62929087/165665268-a95a7f5f-b5c7-43dd-ae76-cc0e66820b2d.png)
